app/config.py

```diff
@@ -79,6 +79,14 @@ class Config(object):
 
     # DB conection string
     SQLALCHEMY_DATABASE_URI = os.getenv('SQLALCHEMY_DATABASE_URI')
+    SQLALCHEMY_DATABASE_READER_URI = os.getenv('SQLALCHEMY_DATABASE_READER_URI')
+    # By making the database reader optional, we can revert to a single writer
+    # instance configuration easily.
+    if SQLALCHEMY_DATABASE_READER_URI:
+        SQLALCHEMY_BINDS = {
+            'writer': SQLALCHEMY_DATABASE_URI,
+            'reader': SQLALCHEMY_DATABASE_READER_URI,
+        }
```

Comment:
I saw there is no logger in that `config` module but wouldn't it make sense to have one? We could display a few info messages on startup to make sure that certain configuration loaded correctly, such in this case as the writer/reader custom load balancing.